src/Type/Numeric.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace GreenTea\Phypes\Type;
+
+use GreenTea\Phypes\Type\Type;
+
+/**
+ * Numeric class validates supplied input is indeed a numeric value.
+ * @author Mangopeaches <https://github.com/mangopeaches>
+ */
+class Numeric extends Type
```

Comment:
The idea of a numeric type seems nice, maybe you could create a different version that complies with the design of the project?